app/assets/javascripts/application.js

```diff
@@ -19,14 +7,17 @@ jQuery(function($) {
   $('#payment-form').submit(function(event) {
     var $form = $(this);
     $form.find('button').prop('disabled', true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
+    if ($(".credit-card")[0].style.display != "none"){
```

Comment:
That was originally set in erb, but I thought it was best to keep JavaScript-y things out of erb files. Is it okay to put it back there?